i18n/en.json

```diff
@@ -1140,6 +1140,8 @@
   "admin.plugins.settings.marketplaceUrl": "Marketplace URL:",
   "admin.plugins.settings.marketplaceUrlDesc": "URL of the marketplace server.",
   "admin.plugins.settings.marketplaceUrlDesc.empty": " Marketplace URL is a required field.",
+  "admin.plugins.settings.requirePluginSignature": "Require Plugin Signature:",
+  "admin.plugins.settings.requirePluginSignatureDesc": "When true, uploading plugins is disabled and plugins are always verified during Mattermost server startup and initialization.",
```

Comment:
If we say that uploading is disabled then we should include how they can still install plugins through marketplace which may not be that obvious if not familiar. 

Maybe something like:

"When true, plugins signatures are verified at installation and server startup. Additionally, plugins may only be installed through the Plugins Marketplace and manual uploading plugins is disabled."

0/5 on exact wording.

Reply:
+1

Reply:
Updated. 
![image](https://user-images.githubusercontent.com/25732808/68060935-d947af00-fcd8-11e9-83f3-29a1654b2ac6.png)

Reply:
Left the link as a generic link for now. Will update when @justinegeffen finalizes the doc.